DanielVargas/tic_tac_toe/src/com/company/Tictactoe.java

```diff
@@ -0,0 +1,176 @@
+package com.company;
+import java.util.Scanner;
+
+public class Tictactoe {
+    final int cols=3;
+    final int rows=3;
+    private String [][] board;
+    private String player1;
+    private String player2;
+    private int num_of_plays;
+    Tictactoe(String player1, String player2) {
+        this.board = new String[this.cols][this.rows];
+        this.player1 = player1;
+        this.player2 = player2;
+        this.num_of_plays=0;
+        for (int i = 0; i < this.rows; i++) {
+            for (int j = 0; j < this.cols; j++) {
+                this.board[i][j] = "";
+            }
+        }
+    }
+    public void printboard()
+    {
+        System.out.println("+--+--+--+");
+        for (int i=0;i<this.rows;i++)
+        {
+            System.out.print("| ");
+            for (int j=0;j<this.cols;j++)
+            {
+                if (this.board[i][j].length()>0)
+                    System.out.print(this.board[i][j]);
+                else
+                    System.out.print(" ");
+                System.out.print("| ");
+            }
+            System.out.println("");
+            System.out.println("+--+--+--+");
+        }
+    }
+    private boolean setplay(int row, int col, String player)
+    {
+        String mark="X";
+        if (player==this.player2)
```

Comment:
player and player1 are String objects, so it's not correct to compare them using == operator, because it is not comparing actual value but locations in memory.
Use "equals()" method for this purpose.